src/components/context/AppActionContext.tsx

```diff
@@ -26,7 +26,7 @@ const AppActionContext =
   createContext<AppActionContextType>(defaultContextValue);
 
 export const AppActionProvider: FC<PropsWithChildren> = ({ children }) => {
-  const appAction = hooks.useAppActions();
+  const appAction = hooks.useAppActions({ enabled: true });
```

Comment:
Mhmmm isn't it enabled by default ? I did the change, maybe I did something wrong...